types/dottie/index.d.ts

```diff
@@ -0,0 +1,194 @@
+// Type definitions for dottie 2.0
+// Project: https://github.com/mickhansen/dottie
+// Definitions by: Dom Armstrong <https://github.com/domarmstrong/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare const dottie: dottie.Dottie;
```

Comment:
Thanks, I tried this but completely overlooked that I was actually using `export const memoizePath` so was getting an error in my tests. `export let` sorted that out ^^. Updated